docs/source/concepts/glossary.rst

```diff
@@ -382,6 +429,17 @@ A **command** is an instruction to add a transaction to the `ledger <#ledger-dam
 
 .. _daml-lf:
 
+Atomic Transaction
+==================
+
+The ability to effect a set of changes atomically such that either all changes happen or nothing happens in case of a conflict or error.
```

Comment:
```suggestion
A set of changes that is committed such that either all of them happen simultaneously, or none at all in case of a conflict or error.
```
Unnecessary circular reference. Unnecessary repetition. However, I don't think that "atomic transaction" should be a glossary definition that we explain. There are far better explanation elsewhere (e.g. Wikipedia).

Reply:
I added this in from the Canton docs. I can however, see the point.  I'll remove.